assets/js/handlebars/templates/versions-dropdown.handlebars

```diff
@@ -9,5 +9,9 @@
         </option>
       {{/each}}
     </select>
+    {{#if jumpToLatestUrl }}
+      <br />
+      <span class="sidebar-projectVersionsLatestLink"><a href="{{jumpToLatestUrl}}">Go to latest</a></span>
```

Comment:
The Rust docs say "go to latest version", not "go to latest"; I realize we have limited space because this is in the sidebar, but I wonder if it might instead be possible to say something like `View latest (v{{latestVersion}})`?